app/helpers/react_helper.rb

```diff
@@ -2,4 +2,19 @@ module ReactHelper
   def react_component(name, props = {})
     content_tag(:div, '', data: { react_component: { name: name, props: props } })
   end
+
+  def react_header_props(current_user)
+    if current_user
+      {
+        username: current_user.first_name,
+        isAdmin: current_user.admin?,
+        editUserLink: edit_user_path(current_user),
+        profileLink: user_path(current_user),
+        adminLink: rails_admin_path,
+        logoutLink: destroy_user_session_path,
+      }
+    else
+      {}
```

Comment:
Use ternary syntax `current_user ? {...} : {}` instead of multiline `if/else`
Or
```ruby
def react_header_props(current_user)
  return {} unless current_user

  {
    username: current_user.first_name,
    isAdmin: current_user.admin?,
    editUserLink: edit_user_path(current_user),
    profileLink: user_path(current_user),
    adminLink: rails_admin_path,
    logoutLink: destroy_user_session_path,
  }
end
```